hw_diag/templates/diagnostics_page_light_miner.html

```diff
@@ -63,9 +63,9 @@ <h3 class="text-center mb-4">Diagnostics Breakdown</h3>
                   {% endif %}
                 </tr>
                 <tr>
-                  <td>Hostname</td>
+                  <td>Hostname (IP Address)</td>
```

Comment:
Minor comment but maybe Local IP would be better?

I guess we could also put the Public IP on diagnostics as well but probably better on a different line?

Reply:
I think we leave it like this for now, and then we can make it look pretty with the UI updates and include both local and external IPs...

Reply:
Sounds like a good plan 👌